psl-core/src/main/java/org/linqs/psl/evaluation/statistics/RankingEvaluator.java

```diff
@@ -21,255 +21,254 @@
 import org.linqs.psl.config.Options;
 import org.linqs.psl.model.atom.GroundAtom;
 import org.linqs.psl.model.predicate.StandardPredicate;
+import org.linqs.psl.model.term.Constant;
+import org.linqs.psl.model.term.UniqueIntID;
 import org.linqs.psl.util.MathUtils;
+import org.linqs.psl.util.RandUtils;
+import org.linqs.psl.util.StringUtils;
+
```

Comment:
Extra newline.
I'm super picky on whitespace issues, so make sure to check all your changes.